### DISCUSSION TITLE
I can help with translation.

### DISCUSSION THREAD

**Discussion by Powiadomienie** [pear-devs/pear-desktop#326] [Ideas]
As said in title i could help translate it into Polish so pepole could navigate more comfortably.

(I've helped in translation of this project https://github.com/hemantapkh/TorrentHunt )

✉ Contact me:
oskarsinger97@gmail.com
! ScriptMaker#7586

Edit:
I noticed that it already has a translation, sorry for making useless discussion.

**Comment by inson1** [comment]
@Powiadomienie No it doesnt :) that was probably component from youtube music. This app doesnt have translations yet :)
I created issue for that #1411.

**Comment by JellyBrick** [comment]
<a href="https://hosted.weblate.org/engage/youtube-music/">
  <img src="https://hosted.weblate.org/widget/youtube-music/i18n/multi-auto.svg" alt="translation status" />
  <img src="https://hosted.weblate.org/widget/youtube-music/i18n/287x66-black.png" alt="translation status 2" />
</a>
